### PR TITLE
updated final

### PR DESCRIPTION
**Pull request by 7ibakana**
this should have no merge conflicts @yp0975eu , @NateBigStone , @ashley7295 , @sunnycake , @cjdufault

### CODE REVIEW

File: lmn/views/views_notes.py [hunk 1]
```diff
@@ -55,6 +55,32 @@ def notes_for_show(request, show_pk):
 def note_detail(request, note_pk):
     note = get_object_or_404(Note, pk=note_pk)
     return render(request, 'lmn/notes/note_detail.html', { 'note': note })
+    @login_required
```
Comment (yp0975eu) [inline lmn/views/views_notes.py:58]: I think this is an indentation error :(

**Review by yp0975eu — CHANGES_REQUESTED** [review]
There's an indentation error in this PR in `views_notes.py`

**Comment by yp0975eu** [comment]
I also am still seeing errors in the tests. 
`FAILED (errors=4)`

File: lmn/views/views_notes.py (hunk 1 above)
Comment (7ibakana) [reply lmn/views/views_notes.py]: lol I don't know how I did that but its fixed...the errors in testing unfortunately I'm not able to figure out where my problem is...I hate testing!

**Pull request merged**